templates/pages/admin/helpdesk_home_custom_scene_config.html.twig

```diff
@@ -0,0 +1,178 @@
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+
+<div class="container-xl ms-0 mt-3">
+    <h2 class="fs-2 mb-2">
+        {{ __("Custom illustrations") }}
+    </h2>
+
+    {% set is_root_entity = entity.getID() == 0 %}
+    {% if is_root_entity %}
+        {% set options = {
+            (constant("Entity::SCENE_DEFAULT")): __("Default illustration"),
+            (constant("Entity::SCENE_CUSTOM")) : __("Custom illustration"),
+        } %}
+    {% else %}
+        {% set options = {
+            (constant("Entity::SCENE_DEFAULT")): __("Default illustration"),
+            (constant("Entity::SCENE_CUSTOM")) : __("Custom illustration"),
+            (constant("Entity::SCENE_INHERIT")): __("Inherited from parent entity"),
+        } %}
+    {% endif %}
+
+    <form method="POST" action="{{ entity.getFormURL() }}">
+        <div class="row mb-3">
+            {% set sides = [
+                {
+                    id: "left_side",
+                    label: __("Left side"),
+                    dropdown_label: __("Left side configuration"),
+                    field: 'custom_helpdesk_home_scene_left',
+                    config: entity.getSceneConfigForDropdown(
+                        'custom_helpdesk_home_scene_left'
+                    ),
+                    default: 'shelves',
+                },
+                {
+                    id: "right_side",
+                    label: __("Right side"),
+                    dropdown_label: __("Right side configuration"),
+                    field: 'custom_helpdesk_home_scene_right',
+                    config: entity.getSceneConfigForDropdown(
+                        'custom_helpdesk_home_scene_right'
+                    ),
+                    default: 'desk',
+                },
+            ] %}
+
+            {% for side in sides %}
+                <section
+                    class="col-6"
+                    aria-labelledby="{{ side.id ~ "heading" }}"
+                >
+                    <h3
+                        id="{{ side.id ~ "heading" }}"
+                        class="mb-2 mt-1"
+                    >{{ side.label }}</h3>
+
+                    {# Main value input dropdown #}
+                    <div class="mb-2">
+                        {% do call('Dropdown::showFromArray', [
+                            side.field,
+                            options,
+                            {
+                                value: side.config,
+                                width: "100%",
+                                aria_label: side.dropdown_label,
+                            }
+                        ]) %}
```

Comment:
There is a `dropdownArrayField` macro.

Reply:
I don't want the full macro, I just need a dropdown.